activestorage/test/service/mirror_service_test.rb

```diff
@@ -61,4 +61,10 @@ class ActiveStorage::Service::MirrorServiceTest < ActiveSupport::TestCase
         @service.url(@key, expires_in: 2.minutes, disposition: :inline, filename: filename, content_type: "text/plain")
     end
   end
+
+  test "path for file in all mirror services" do
+    SERVICE.mirrors.each do |mirror|
+      assert mirror.path_for(@key)
+    end
+  end
```

Comment:
This test passes without the code change. We want to test that the mirror delegates `path_for` to the primary:

```ruby
test "path for file in primary service" do
  assert_equal @service.primary.path_for(@key), @service.url_for(@key)
end
```